go.mod

```diff
@@ -11,9 +11,10 @@ require (
 	k8s.io/apimachinery v0.19.2
 	k8s.io/client-go v0.19.2
 	k8s.io/code-generator v0.19.2
-	k8s.io/gengo v0.0.0-20200728071708-7794989d0000 // indirect
 	k8s.io/klog/v2 v2.3.0 // indirect
 	k8s.io/kube-openapi v0.0.0-20200831175022-64514a1d5d59 // indirect
 	sigs.k8s.io/controller-runtime v0.6.2
 	sigs.k8s.io/controller-tools v0.4.0
 )
+
+replace github.com/ahmetb/gen-crd-api-reference-docs => github.com/jpeach/gen-crd-api-reference-docs v0.2.1-0.20201214045921-2511762f1bee
```

Comment:
It seems like Ahmet has already approved your PR upstream. Can we wait till it gets merged and then continue to reference upstream rather than a fork here?
The fork _could_ get out of date.

Reply:
Yeh I don't know what the process is or how long it might take to land that patch. My intention (and preference) is to unfork once it lands.